### ISSUE TITLE
Unknown prop `date` on <div> tag

### ISSUE THREAD

**Issue by edidiway** [dmtrKovalenko/material-ui-pickers#16]
hi, I get this warning when use the library.

Unknown prop `date` on `<div>` tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop

I use this component as a component inside redux-form

MyForm.tsx
`<Field name="DOB" component={DatePickerGroup} />`

DatePickerGroup.tsx
```
import * as React from 'react';
import { findDOMNode } from 'react-dom';
import { DatePicker } from 'material-ui-pickers';
import * as moment from 'moment';
import { WrappedFieldProps } from 'redux-form';

export namespace DatePickerGroup {
    export interface AllProps extends OwnProps, WrappedFieldProps {
        
    }
    export interface OwnProps {
        id?: string,
        label?: string,
        className?: string,
        shrink?: boolean,
        fullWidth?: boolean,
        margin?: string,
        disabled?: boolean,
        readOnly?: boolean
    }
    export interface State {
        
    }
}

class DatePickerGroup extends React.Component<DatePickerGroup.AllProps, DatePickerGroup.State> {
    constructor(props) {
        super(props);
    }

    private inputControl: HTMLInputElement;

    handleDateChange = (date: moment.Moment) => {
        this.props.input.onChange(date.format("YYYY-MM-DD"));
    }

    render() {
        const {
            input, id, label, className, shrink, fullWidth, margin, disabled, readOnly,
            meta: { touched, error }
        } = this.props
        const selectedDate = moment(input.value, "YYYY-MM-DD").isValid() ? moment(input.value, "YYYY-MM-DD") : moment(new Date());

        return (
            <div>
                <DatePicker
                    format="DD/MM/YYYY"
                    value={selectedDate}
                    onChange={this.handleDateChange}
                />
            </div>
        )
    }
}

export default DatePickerGroup;
```

I use:
react: 15.6.2
typescript: 2.5.3,
material-ui: 1.0.0-beta.18
redux-form: 7.0.4

**Comment by dmtrKovalenko** [comment]
Can you please provide example of code

**Comment by edidiway** [comment]
hi, I have updated the question, it was included the example.

**Comment by dmtrKovalenko** [comment]
But it works fine, isnt it?

**Comment by edidiway** [comment]
yes, it works fine.. but usually we don't like a warning / error.

here is the screenshot of the warning
![capture](https://user-images.githubusercontent.com/3364840/32160978-0f4442c0-bda8-11e7-97b0-aef0ca240120.PNG)

**Comment by dmtrKovalenko** [comment]
Yeah, I`m sorry, I`ll fix it in the next release, probably today. 
Its really strange that I dont have such warning in my demo project.

**Comment by edidiway** [comment]
do you use the same react version / material-ui?

anyway, another input for this library, when we had choose the year value, it is better the selection redirect back to the date selection. 
Current, I have manually click on header of "date".

Thank you.

**Comment by dmtrKovalenko** [comment]
Should be fixed in [v1.0.0-alpha.8](https://github.com/dmtrKovalenko/material-ui-pickers/releases/tag/v1.0.0-alpha.8)
